SU2_CFD/src/output/COutput.cpp

```diff
@@ -130,6 +130,18 @@ COutput::COutput(const CConfig *config, unsigned short ndim, bool fem_output):
   cauchyValue = 0.0;
   convergence = false;
 
+  /*--- Prepare windowedAverages ---*/
+  if (config->GetTime_Domain()) {
+    for (const auto it : historyOutput_Map) {
```

Comment:
Sorry my bad, gave you an incorrect suggestion.
```suggestion
    for (const auto& it : historyOutput_Map) {
```